research/python/scripts/run_backtest_jan2020.py

```
@@ -0,0 +1,523 @@
+
+    Fetches NIFTY data from jugaad-data provider and derives:
+    - Intraday price movements
+    - Volatility from high-low ranges
+    - OI decay patterns from derivatives data
+    - Regime classification based on market microstructure
+    """
+
+    logger.info("Fetching real market data for January 2020...")
+
+    # Initialize data provider
+    provider = GalactusDataProvider()
```

Ruff (0.14.10)

Local variable `provider` is assigned to but never used

Remove assignment to unused variable `provider`

(F841)